app/stacks/uk-west/applications-service/terragrunt.hcl

```diff
@@ -59,6 +60,7 @@ inputs = {
   back_office_service_bus_nsip_project_update_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_project_update_topic_id
   back_office_service_bus_nsip_representation_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_representation_id
   back_office_service_bus_nsip_exam_timetable_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_exam_timetable_id
+  back_office_service_bus_nsip_advice_topic_id                  = dependency.back_office_uks.outputs.service_bus_nsip_s51_advice_topic_id
```

Comment:
This need to be 
dependency.back_office_ukw.outputs.service_bus_nsip_s51_advice_topic_id